diff --git ./third_party/crashpad/crashpad/util/linux/ptracer.cc ./third_party/crashpad/crashpad/util/linux/ptracer.cc
index 24ef002d3f5c..f965805a1b41 100644
--- ./third_party/crashpad/crashpad/util/linux/ptracer.cc
+++ ./third_party/crashpad/crashpad/util/linux/ptracer.cc
@@ -69,9 +69,10 @@ bool GetThreadArea32(pid_t tid,
                      bool can_log) {
   size_t index = (context.t32.xgs & 0xffff) >> 3;
   user_desc desc;
-  if (ptrace(
-          PTRACE_GET_THREAD_AREA, tid, reinterpret_cast<void*>(index), &desc) !=
-      0) {
+  if (ptrace(static_cast<enum __ptrace_request>(PTRACE_GET_THREAD_AREA),
+             tid,
+             reinterpret_cast<void*>(index),
+             &desc) != 0) {
     PLOG_IF(ERROR, can_log) << "ptrace";
     return false;
   }
